modules/network/README.md

```diff
@@ -0,0 +1,110 @@
+| [aws_route.private_nat_gateway_route](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/route) | resource |
+| [aws_route.public_internet_gateway_route](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/route) | resource |
+| [aws_route_table.private_route_table](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/route_table) | resource |
+| [aws_route_table.public_route_table](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/route_table) | resource |
+| [aws_route_table_association.private_subnet_association](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/route_table_association) | resource |
+| [aws_route_table_association.public_subnet_association](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/route_table_association) | resource |
+| [aws_subnet.private_subnet](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/subnet) | resource |
+| [aws_subnet.public_subnet](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/subnet) | resource |
+| [aws_vpc.arc_vpc](https://registry.terraform.io/providers/hashicorp/aws/5.48.0/docs/resources/vpc) | resource |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| <a name="input_availability_zones"></a> [availability\_zones](#input\_availability\_zones) | List of availability zones | `list(string)` | n/a | yes |
+| <a name="input_aws_region"></a> [aws\_region](#input\_aws\_region) | n/a | `string` | `"us-east-1"` | no |
+| <a name="input_private_subnet_cidrs"></a> [private\_subnet\_cidrs](#input\_private\_subnet\_cidrs) | CIDR blocks for private subnets | `list(string)` | n/a | yes |
+| <a name="input_public_subnet_cidrs"></a> [public\_subnet\_cidrs](#input\_public\_subnet\_cidrs) | CIDR blocks for public subnets | `list(string)` | n/a | yes |
+| <a name="input_vpc_cidr"></a> [vpc\_cidr](#input\_vpc\_cidr) | CIDR block for the VPC | `any` | n/a | yes |
+| <a name="input_vpc_name"></a> [vpc\_name](#input\_vpc\_name) | Name for the VPC | `any` | n/a | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| <a name="output_eip_ids"></a> [eip\_ids](#output\_eip\_ids) | The IDs of the created Elastic IPs for NAT gateways |
+| <a name="output_nat_gateway_ids"></a> [nat\_gateway\_ids](#output\_nat\_gateway\_ids) | The IDs of the created NAT gateways |
+| <a name="output_private_subnet_ids"></a> [private\_subnet\_ids](#output\_private\_subnet\_ids) | The IDs of the created private subnets |
+| <a name="output_public_subnet_ids"></a> [public\_subnet\_ids](#output\_public\_subnet\_ids) | The IDs of the created public subnets |
+| <a name="output_vpc_id"></a> [vpc\_id](#output\_vpc\_id) | The ID of the created VPC |
+<!-- END_TF_DOCS -->
```

Comment:
`<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->`

Reply:
fixed